src/index.ts

```diff
@@ -107,8 +107,8 @@ async function authorize(
 		body: JSON.stringify(payload),
 	});
 	const body = await res.json();
-	if (!res.ok) {
-		throw new AuthorizeError(body.Message);
+	if (!res.ok || body === '00000000-0000-0000-0000-000000000000') {
```

Comment:
I wonder if we should make this be more flexible and if body comes back matching /^[0-]*$/ we treat that has a failure?

Reply:
What do you think about this?  I can merge it after this change, or let me know if you don't think we should make this change.